### ISSUE TITLE
Add support for Swedish calendar

### ISSUE THREAD

**Issue by elyoh** [FamilySearch/GEDCOM#28]
This proposal defines the Swedish calendar as follows:

`SWEDISH`

The Swedish calendar was introduced by the Government of Sweden on 1700 as part of a plan to switch from the Julian calendar to the Gregorian calendar.  It was intended to omit all 11 Julian leap days in the period 1700 to 1740 so that the Swedish calendar gradually aligned with the Gregorian calendar, which at the time, was 11 days ahead of the Julian calendar.  Accordingly, the Julian leap day in 1700 was omitted leaving the calendar one day ahead of the Julian calendar.  Julian leap days in 1704 and 1708 were not omitted.  In 1712, the Swedish calendar was abandoned.  The omitted leap day was reinseted as 30 Feb in 1712, realigning with the Julian calendar. 

This calendar uses the same months as the Gregorian calendar, differing only in the structure of the month of February which can have 28, 29, or 30 days.

| Year | Days in Feb (Julian) | Days in Feb (Swedish)|Note|
|---|---|---|---|
| 1700  | 29  | 28  | Leap day omitted in 1700|
| 1704  | 29  | 29 |  |
| 1708  | 29  | 29 |  |
| 1712  | 29  | 30 | The leap day omitted from 1700 is reinserted as 30 Feb |

This calendar was in use from Swedish day 1 Mar 1700 to 30 Feb 1712. Thus, no epoch marker is permitted in this calendar.

The URI for this calendar is `g7:cal-SWEDISH`

**Comment by BertKoor** [comment]
Yes please. If it were implemented as an extension, then (if rules are strictly applied) it must use `extTag` for all month names, which would be a pity.

https://github.com/FamilySearch/GEDCOM/blob/4fd1d1af98d2ca4986577b5cd1e148973f37d99c/specification/gedcom-2-data-types.md?plain=1#L113

**Comment by tychonievich** [comment]
Discussed in steering committee

- We agree that using standard tags for standard months inside an extension calendar is OK, and made a separate issue (#797) to track adjusting that text in the spec.
    - Because Julian and Gregorian use the same standard month tag for months that have different numbers of days, we think that it is OK to keep doing that, including using `FEB` in the Swedish calendar even for `30 FEB 1712` which would not make sense in other calendars that use `FEB`.
- Adding additional calendars remains a 7.1 topic of interest.
- We discussed if the gradual change between Julian and Gregorian warranted a new calendar, and decided it did because there were dates during the transition that were not on either calendar. We looked for other examples of similar gradual changes but did not find any; if others know of them, please let us know.
- We think that as there have been requests for this calendar from multiple users, it may be ready for inclusion in the v7.1 branch. We should review other calendar proposals to see if they are similarly ready.
